aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -1065,7 +1065,7 @@ void embedding_bag_cpu_max_out(
     const Tensor& indices,
     const Tensor& offset2bag,
     const Tensor& output,
-    bool include_last_offset,
+    bool /*include_last_offset*/,
```

Comment:
`[[maybe_unused]]` is part of C++17 standard, but not associating name with the argument is not part of the standard, so it might not work with VC++
```suggestion
    [[maybe_unused]] bool include_last_offset,
```